pkg/virt-controller/services/template_test.go

```diff
@@ -2930,6 +2930,10 @@ var _ = Describe("Template", func() {
 			vmi := v1.VirtualMachineInstance{ObjectMeta: metav1.ObjectMeta{
 				Name: "testvmi", Namespace: "default", UID: "1234",
 			}}
+
+			networkClient := fakenetworkclient.NewSimpleClientset()
+			virtClient.EXPECT().NetworkClient().Return(networkClient).AnyTimes()
```

Comment:
Because they run this [function](https://github.com/kubevirt/kubevirt/blob/b2887836ea060972a33782f3708c6bdbafb7c71e/pkg/virt-controller/services/template_test.go#L113) which has this lines (line 138)